.github/workflows/ruby.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [jruby-9.2.12.0, 2.4, 2.5, 2.6, 2.7]
+        ruby: [jruby-9.3.2.0, 2.7, 3.0]
```

Comment:
I believe you have to use `'3.0'` or yaml resolves this to the integer `3`, which will then test on `3.1`